pkgs/development/libraries/qcoro/default.nix

```diff
@@ -2,29 +2,33 @@
 , mkDerivation
 , fetchFromGitHub
 , cmake
+, gcc11
 , libpthreadstubs
 , qtbase
+, qtwebsockets
 }:
 
 mkDerivation rec {
   pname = "qcoro";
-  version = "0.4.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "danvratil";
     repo = "qcoro";
     rev = "v${version}";
-    sha256 = "sha256-RVpyL+BklX8Wyk9Xj9UyuvNK5Vev8ZsrOSMxX1HtcHU=";
+    sha256 = "sha256-6kRWBzspwsO0Q6/8gQUr69DJjmkPa3lWrKTmSgVn6V4=";
   };
 
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [
     cmake
+    gcc11
```

Comment:
Please use gcc11Stdenv

Reply:
This derivation is using `mkDerivation` from `qt5` so that it can provide things like `wrapQtAppsHook`. I can't just drop in `gcc11Stdenv.mkDerivation`. How do you think I should use `gcc11Stdenv`?

Reply:
qt5 now works with stdenv.mkDerivation and should be used all the time. I am not sure if it works with gcc11Stdenv. Can you please try it?